etna/media/blocks.py

```diff
@@ -3,34 +3,23 @@
 from django.template.loader import render_to_string
 
 from wagtail.core import blocks
-from wagtailmedia.blocks import ChooserBlock, AbstractMediaChooserBlock
+from wagtail.images.blocks import ImageChooserBlock
+from wagtailmedia.blocks import AbstractMediaChooserBlock
 
 
 class EtnaMediaBlock(AbstractMediaChooserBlock):
     def render_basic(self, value, context=None):
-        if not value:
-            return ""
+        pass
 
-        context = {
-            "value": value,
-            "src": value.sources[0]["src"],
-            "type": value.sources[0]["type"],
-        }
 
-        # Check for empty rich text fields.
-        if value.description == "<p></p>":
-            value.description = None
-
-        if value.transcript == "<p></p>":
-            value.transcript = None
-
-        # Render using the appropriate template.
-        if value.type == "audio":
-            return render_to_string("media/blocks/media-block--audio.html", context)
-        elif value.type == "video":
-            return render_to_string("media/blocks/media-block--video.html", context)
-        else:
-            return ""
+class MediaBlock(blocks.StructBlock):
+    heading = blocks.CharBlock(max_length=100, required=True)
```

Comment:
As discussed, would you mind removing the heading field?

Reply:
Done